src/search-injector/background/index.js

```diff
@@ -0,0 +1,24 @@
+import { makeRemotelyCallable } from 'src/util/webextensionRPC'
+import * as searchInjector from './interface'
+import * as constants from '../constants'
+
+makeRemotelyCallable({
+    fetchSearchInjection: searchInjector.fetchSearchInjection,
+    toggleSearchInjection: searchInjector.toggleSearchInjection,
+})
+
+// Temporary placement of this functions
+browser.runtime.onMessage.addListener(value => {
+    switch (value.action) {
+        case 'openOverviewPagewithParams':
+            return openOverviewPagewithParams(value.queryParams)
+        default:
+            break
+    }
+})
+
+async function openOverviewPagewithParams(queryParams) {
```

Comment:
There's a `openOverview` function in `src/background` module. Maybe you could integrate the `queryParams` param into that one and import here for re-use.